docker-compose.yml

```
@@ -24,6 +25,17 @@ services:
       - ./db/data:/pb/pb_data
     profiles:
       - dev
+  sports-db:
+    build:
+      context: ./db
```

This will build the Dockerfile with the current directory as `db`, which isn't right.

```suggestion
      dockerfile: ./db/Dockerfile
      context: ./sports_db
```